psl/psl-core/src/builtin_connectors/mssql_datamodel_connector.rs

```diff
@@ -262,7 +262,7 @@ impl Connector for MsSqlDatamodelConnector {
     ) {
         if let ast::SchemaPosition::Model(
             _model_id,
-            ast::ModelPosition::Field(_, ast::FieldPosition::Attribute("default", _, None)),
```

Comment:
changed to line-up with changes in find_at_position

Reply:
Ah, this should be `AttributePosition::Atribute`

Reply:
Fixed :)

Reply:
Can we test this?

Reply:
Ah, reading comprehension is hard. This is exclusively used in the `@default` attribute. A test has been added to ensure that this is only offered when not in something else. However, I will note that because we offer this completion here, the completions that are also offered in language-tools here get obfuscated. This is both not new nor in scope for this PR but yeah

<img width="586" alt="image" src="https://github.com/prisma/prisma-engines/assets/29753584/88c63fe6-ffad-43ac-9fd6-5c370eba640c">

<img width="521" alt="image" src="https://github.com/prisma/prisma-engines/assets/29753584/be8507e4-9e59-40f0-883d-27fb2768ba71">